src/common/low_precision_transformations/src/recurrent_cell.cpp

```diff
@@ -36,13 +29,14 @@ RecurrentCellTransformation::RecurrentCellTransformation(const Params& params) :
     const auto fq_W = wrap_fake_quantize(W);
     const auto fq_R = wrap_fake_quantize(R);
 
-    const auto dequantization_X = wrap_dequantization(ov::pass::pattern::any_input(), true);
+    dequantization_X = wrap_dequantization(ov::pass::pattern::any_input(), true);
     const auto dequantization_H = wrap_dequantization(ov::pass::pattern::any_input(), true);
     const auto dequantization_W = wrap_dequantization(ov::pass::pattern::any_input(), true);
     const auto dequantization_R = wrap_dequantization(ov::pass::pattern::any_input(), true);
 
     const auto dequantization_without_subtract_X = wrap_dequantization(ov::pass::pattern::any_input(), false);
-    const auto dequantization_without_subtract_H = wrap_dequantization(ov::pass::pattern::any_input(), false);
+    const auto dequantization_without_subtract_H_u8 = wrap_dequantization(ov::pass::pattern::any_input(pattern::type_matches(element::u8)), false);
+    dequantization_without_subtract_H_i8 = wrap_dequantization(ov::pass::pattern::any_input(pattern::type_matches(element::i8)), false);
```

Comment:
Are we sure this limitation should be introduced to this transformation and not "convert_fq_rnn_to_quantized_rnn.cpp"? Because precision limitation is more about cpu plugin limitations and decision, not LPT.

Reply:
Done